File: tests/indexing/parsing/lang/fixtures/python/Conditionals.py
def categorize(score, bonus):
    label = ""
    if score >= 90:
        label = "excellent"
        score = score - bonus
    elif score >= 70:
        base = score - 70
        label = f"good ({base})"
    elif score >= 50:
        label = "average"
    else:
        label = "below average"
        score = 0
        bonus = 0
    return label, score


def clamp(value, lo, hi):
    adjusted = value
    if value < lo:
        deficit = lo - value
        adjusted = lo + deficit // 2
    elif value > hi:
        adjusted = hi
        return adjusted - 1
    return adjusted

File: tests/indexing/parsing/lang/fixtures/elixir/Conditionals.ex
defmodule Conditionals do
  def categorize(score) do
    if score >= 90 do
      tier = "gold"
      bonus = score - 90
      {tier, bonus}
    else
      tier = "bronze"
      penalty = 90 - score
      {tier, penalty}
    end
  end

  def clamp(value, lo, hi) do
    if value < lo do
      deficit = lo - value
      adjusted = lo + div(deficit, 2)
      adjusted
    end

    unless value > hi do
      surplus = value - hi
      capped = hi - abs(surplus)
      capped
    end
  end
end
